rules/falco_rules.yaml

```diff
@@ -3397,6 +3397,29 @@
   output: "File execution detected from /dev/shm (proc.cmdline=%proc.cmdline connection=%fd.name user.name=%user.name user.loginuid=%user.loginuid container.id=%container.id evt.type=%evt.type evt.res=%evt.res proc.pid=%proc.pid proc.cwd=%proc.cwd proc.ppid=%proc.ppid proc.pcmdline=%proc.pcmdline proc.sid=%proc.sid proc.exepath=%proc.exepath user.uid=%user.uid user.loginname=%user.loginname group.gid=%group.gid group.name=%group.name container.name=%container.name image=%container.image.repository)"
   priority: WARNING
   tags: [host, container, mitre_execution, mitre_defense_evasion, T1036.005, T1059.004]
+  
+- list: other_utilities
+  items: [awk, sed, grep, python, perl, bash, cron, systemd-timer, atd]
+  
+- rule: Execute Command Via Utility
+  desc: Detects execution of commands via utilities commonly used to parse text, scripting languages, and system utilities
+  condition: >
+    (evt.type = execve and
```

Comment:
There are multiple ways a process may be spawned, and we should take that into account. The macro `spawned_process` should be more accurate.